jupyter_matlab_proxy/app.py

```diff
@@ -28,7 +28,13 @@
     from .gui.static import js
     from .gui.static import media
 
-logging.basicConfig(level=logging.INFO)
+logLevel=logging.WARNING
+if os.environ.get("LOG_LEVEL") is not None:
+    lvl = logging._nameToLevel.get(os.environ.get("LOG_LEVEL").upper())
```

Comment:
Using private fields is not recommended.
Two other approaches:
https://stackoverflow.com/a/60679407
https://stackoverflow.com/a/35689599